test/sanity/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# pip packages required to build sanity tests
```

Comment:
We already have a sanity requirements list:

https://github.com/ansible/ansible/blob/devel/test/runner/requirements/sanity.txt